src/container/srv_container.c

```diff
@@ -517,6 +517,8 @@ cont_destroy_bcast(crt_context_t ctx, struct cont_svc *svc,
 	uuid_copy(in->tdi_uuid, cont_uuid);
 
 	rc = dss_rpc_send(rpc);
+	if (rc == 0 && DAOS_FAIL_CHECK(DAOS_CONT_DESTROY_FAIL_CORPC))
+		rc = -DER_TIMEDOUT;
```

Comment:
Since when a CoRPC fails due to timeouts or group version mismatches, it may have been executed successfully on some servers. The rational here is to test retrying the CoRPC and see if anything breaks.